### PR TITLE
Adds README for app:twitter

### PR DESCRIPTION
**Pull request by zakkak**
This PR adds as much info as I could infer from the code. There are some TODOs
left in the readme for hacks, configuration etc. since I felt they were out of
my reach.

### CODE REVIEW

**Comment by A6GibKm** [comment]
I suggest that you add install methods for the prerequisites in said section. Something reasonable would be `apt` command + the command you would use in your distro if this uses other method.

File: modules/app/twitter/README.org [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
++ Retweet
++ Follow and un-follow users
++ Favorite tweets
+
+** Module Flags
+This module provides no flags.
+
+** Plugins
+[[https://github.com/hayamiz/twittering-mode][twittering-mode]]
+[[https://github.com/abo-abo/avy][avy]]
```
Comment (hlissner) [inline modules/app/twitter/README.org]: These should be list items, e.g.

```
+ [[https://github.com/hayamiz/twittering-mode][twittering-mode]]
+ [[https://github.com/abo-abo/avy][avy]]
```

**Comment by zakkak** [comment]
I am thinking that maybe we need a file documenting how to install external dependencies and then linking to that. This way we don't have to replicate instructions on getting gzip etc. on each module that uses them. Same could be done for building Emacs with ImageMagick support.

**Comment by A6GibKm** [comment]
I think it is not bad to assume for the time being that the user is in a distro that comes with gnupg. Most of the stuff left amounts just to a single `apt,dnf,pacman,...`command. While there might be duplicates I think that it would be even more convoluted to try to deduplicate that.

**Comment by zakkak** [comment]
relates to #1166

**Comment by zakkak** [comment]
Regarding gnupg @hlissner this module just needs it for symmetrical encryption (no need for any fancy setup).

**Pull request merged**